overlord/snapstate/backend_test.go

```diff
@@ -146,6 +140,18 @@ func (f *fakeStore) SnapInfo(spec store.SnapSpec, user *auth.UserState) (*snap.I
 		Confinement: confinement,
 		Type:        typ,
 	}
+	switch spec.Channel {
+	case "channel-for-devmode":
+		info.Confinement = snap.DevModeConfinement
+	case "channel-for-classic":
+		info.Confinement = snap.ClassicConfinement
+	case "channel-for-payed":
+		info.Prices = map[string]float64{"USD": 0.77}
```

Comment:
Hopefully no real financial operations are done with money expressed as float :)

Reply:
I too was surprised to find out that yes, some of these things are done with floats. Not AFAIK in aggregate though.